advanced_functionality/multi_model_tensorflow/tf2-MME-regression.ipynb

```diff
@@ -0,0 +1,1534 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    boston = datasets.load_boston()
</pre><p>Boston housing dataset cannot be used. I recommend the California housing dataset instead, which is hosted by AWS on S3: <code>s3://sagemaker-sample-files/datasets/tabular/california_housing/cal_housing.tgz</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2887/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>